### ISSUE TITLE
Menu elements produce warnings based on containing li elements

### ISSUE THREAD

**Issue by epvanhouten** [dequelabs/axe-core#3411]
Product: axe Linter

Expectation: The following markup should not warn:

```
<menu>
  <li>foo</li>
</menu>
```

Actual: The `li` element produces a warning due to not being in a `ul` or `od` element.

Motivation: The [li element](https://html.spec.whatwg.org/multipage/grouping-content.html#the-li-element) is legal in `ol`, `ul`, and `menu`.

Generated by linter included in Microsoft Edge 99.

**Comment by straker** [comment]
Thanks for the issue. I believe this was fixed in our [4.4.0 release](https://github.com/dequelabs/axe-core/releases/tag/v4.4.0) at the end of January, specifically https://github.com/dequelabs/axe-core/pull/3286. My guess is that both Lighthouse and Microsoft Edge haven't had the chance to upgrade to the latest version.

Testing in latest version of axe-core shows no errors or warnings, so going to close.

**Comment by jarrodek** [comment]
I use `axe-core` 4.4.1 via Axe DevTools and this issue is still reproducible. 

The DevTools gives the following description:

Issue Description
```
Ensures <li> elements are used semantically
```

Element location
```
(trimmed) menu > .list-item[data-kind="Core#HttpHistory"]
```

Element source
```html
<li class=" list-item " data-key="fjExMzY0ODM5Mzc3NTI2MTY1ODAwMn4yMDIyLTA0LTI3VDIyOjE0OjAzLjg3Mlp-" data-kind="Core#HttpHistory">
```

To solve this issue, you need to...

Fix the following:

```
List item does not have a <ul>, <ol> parent element without a role, or a role="list".
```

**Comment by jarrodek** [comment]
I am sorry, I just realized that I am building a `<menu>` with sub-menus. This issue is related to me setting the `role="group"` on a sub-menu. This caused the report.
